### PR TITLE
changes required for android Q

### PR DESCRIPTION
**Pull request by bobzilladev**
do not merge, will wait for Q release candidates so api's are locked down.

### CODE REVIEW

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/MainActivity.java [hunk 1]
```diff
@@ -350,12 +351,26 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     private void pieScanningSettings(final SharedPreferences prefs) {
```
Comment (rksh) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/MainActivity.java]: maybe a more descriptive name (pieAndUp?)

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/NetworkActivity.java [hunk 2]
```diff
@@ -361,44 +361,22 @@ public void onMapReady(final GoogleMap googleMap) {
...
         }
 
-        final RelativeLayout rlView = (RelativeLayout) findViewById( R.id.netmap_rl );
+        final RelativeLayout rlView = findViewById( R.id.netmap_rl );
         rlView.addView( mapView );
     }
 
     private void setupButtons( final Network network ) {
         final SharedPreferences prefs = getSharedPreferences(ListFragment.SHARED_PREFS, 0);
-        final Button connectButton = (Button) findViewById( R.id.connect_button );
```
Comment (rksh) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/NetworkActivity.java]: why not surround connect tool w/ a version flag check?

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/MainActivity.java (hunk 1 above)
Comment (bobzilladev) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/MainActivity.java]: it really is just pie, the bad one

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/NetworkActivity.java (hunk 2 above)
Comment (bobzilladev) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/NetworkActivity.java]: it changes based on target sdk, not host sdk, unfortunately.

Comment (rksh) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/NetworkActivity.java]: !! asspandas.

**Pull request merged**